### PR TITLE
[ENG-3885] Update badge alt text

### PR DESCRIPTION
**Pull request by futa-ikeda**
-   Ticket: [ENG-3885]
-   Feature flag: n/a

## Purpose
- Fix a11y issue for badge's alt text: "Alternative text of images should not be repeated as text"

## Summary of Changes
- Update resource card badge image's alt text

## Screenshot(s)
- NA
## Side Effects
- None

## QA Notes

[ENG-3885]: https://openscience.atlassian.net/browse/ENG-3885?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by brianjgeiger — APPROVED** [review]

File: lib/osf-components/addon/components/resources-list/resource-card/template.hbs [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <p data-test-resource-card-type local-class='resourceType'>
         <img
             src={{this.resourceIcon}}
-            alt={{this.resourceTypeName}}
+            alt={{concat this.resourceTypeName ' ' (t 'osf-components.resources-list.badge')}}
```
Comment (adlius) [inline lib/osf-components/addon/components/resources-list/resource-card/template.hbs:5]: Does it make more sense for the translation string to take an argument than using the `{{concat}}` helper here?

**Review by adlius — COMMENTED** [review]
One small nitpick. Consider it approved.

File: lib/osf-components/addon/components/resources-list/resource-card/template.hbs (hunk 1 above)
Comment (futa-ikeda) [reply lib/osf-components/addon/components/resources-list/resource-card/template.hbs:5]: That's a good point! Some languages may put the "badge" descriptor before the resource type 🌍

File: lib/osf-components/addon/components/infinite-list/template.hbs [hunk 2]
```diff
@@ -4,7 +4,9 @@
         {{yield item to='item'}}
     {{/each}}
     {{#if @isLoading}}
-        <LoadingIndicator @dark={{true}} />
+        <li>
```
Comment (futa-ikeda) [inline lib/osf-components/addon/components/infinite-list/template.hbs:7]: Wrapped this in `<li>` as the parent `<ul>` should only have list items as direct children

**Pull request merged**